packages/gallery/src/components/gallery/proGallery/galleryContainerNew.js

```diff
@@ -149,6 +148,7 @@ export class GalleryContainer extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
+    this.getMoreItemsIfNeeded(0);
```

Comment:
call getMoreItemsIfNeeded in case of when on new props this way we have the real container after the first run with the defaults in case of unknown dimensions